projects/rero/ng-core/src/lib/record/editor/editor.service.ts

```diff
@@ -59,21 +59,34 @@ export class EditorService {
     this._hiddenFieldsSubject.next(this._hiddenFields);
   }
 
+  /**
+   * Check if the field is a candidate for the field navigation.
+   * @param field - FormlyFieldConfig, form config to be added
+   * @returns boolean, true if the field is at the root JSONSchema.
+   */
+  isFieldRoot(field: FormlyFieldConfig) {
+    return field.parent.parent.parent === undefined;
```

Comment:
Is there a possibility that field.parent.parent is undefined and crash this condition?

Reply:
Ad I understood no, but we can add some checks.